### DISCUSSION TITLE
o.Slice is not a function

### DISCUSSION THREAD

**Discussion by JamesLloydGranify** [aoe4world/explorer#22] [Bugs / Problems] [answered]
When clicking on the tower from the explorer (Stone, outpost, or toll outpost), I get the titled error.

**Comment by robertvanhoesel** [comment, marked as answer]
Thanks for reporting!

**Reply by robertvanhoesel** [reply]
Fixed in 3fa937da1cb3a2453eaf5f1d54e841a9d2906078
